types/bitcore-lib/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for bitcore-lib 0.15
+// Type definitions for bitcore-lib 8.25
```

Comment:
There are probably a lot of API changes from v0.15 to v8.25 that won't be included in this PR